bin/runtime/src/lib.rs

```diff
@@ -345,50 +335,89 @@ impl_opaque_keys! {
 }
 
 parameter_types! {
-    pub const Offset: u32 = 0;
+    pub const MillisecsPerBlock: u64 = DEFAULT_MILLISECS_PER_BLOCK;
```

Comment:
I think there should be some kind of a warning in the code, that: 
Once you start a chain with a runtime that has given values of these two parameters, then they must stay so forever. If you update to a runtime with different values, then I think they will be ignored, but I'm not super confident -- something unexpected might as well happen.

Reply:
I don't think this is a new thing because of this change, but this deserved a warning a long time ago already...

Reply:
I believe that after an upgrade the new version would take place. Why do you think otherwise?